jp/components/dialog.md

```diff
@@ -1,99 +1,183 @@
 ﻿---
-title: Dialog Window コンポーネント
-_description: Ignite UI for Angular Dialog Window コンポーネントを使用すると、情報ダイアログまたはデータ変換ウィンドウを作成し、リアルタイムで情報を表示して管理できます。
-_keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェット, web ウィジェット, UI ウィジェット, Angular, ネイティブ Angular コンポーネント スィート, ネイティブ Angular コントロール, ネイティブ Angular コンポーネント ライブラリ, Angular Dialog Window コンポーネント, Angular Dialog Window コントロール
-_language: ja
```

Comment:
Did we stop using this metadata field?

Reply:
I noticed none of the topics has this field, may be @simeonoff know more?

Reply:
These are required fields, each article should include them. The values for _description and _keywords are provided by marketing. Simply put - you shouldn't be deleting them.

Reply:
@simeonoff I meant particularly the `_language` metadata field.  That is the one that is being removed in some of the new topics.  I think it's a copy/paste error as I've noticed the developer copying the English topic into the Japanese file.  I have been fixing this as I add the Japanese translations but it's error prone as I have to remember to do it.

Reply:
Not sure if it needs to be in there for the English topic.  Mostly interested if it is still used in the Japanese topics to know if I should continue adding it in there.